README.md

```diff
@@ -1 +1,38 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse - Hello Microverse
+
+This is a simple hello Microverse project that shows how to set up linters and use github workflows.
+
+
+![Screenshot](hello.png)
+
+
+# Built With
+
+- HTML and CSS
+Introducing Linters
+- Sytle Lint
+- WebHint
+- Lighthouse
+
+👤 Authors
+
+👤Kirezi Emery Herve
+- GitHub: [@Emeryherve](https://github.com/Emeryherve)
+# 🤝Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page. Show your support
+
+Give a ⭐️ if you like this project!
+
+# Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+# 📝 License 
+This project is MIT licensed.
```

Comment:
- Great Job for fulfilling most of the functionality, I think you forget to use [this template](https://github.com/microverseinc/readme-template) for your readme,  so please update using the [template](https://github.com/microverseinc/readme-template) to have professional readme 👍 
- check those guidelines when you using the  [template](https://github.com/microverseinc/readme-template) 👇
    - The only optional section is FAQ.
    - Delete the live demo section if you didn't deploy the application.
    - Remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with your custom content.
    - Remove the template logo or replace it with your own logo if you have one.